src/annotator/index.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
-const configFrom = require('./config/index');
-
 const $ = require('jquery');
 
-// Applications
+// Load polyfill for :focus-visible pseudo-class.
+require('focus-visible');
```

Comment:
Can you help me understand why this polyfill becomes necessary because of these changes (i.e. why it wasn't necessary before for the `outline-on-keyboard-focus` mixin)?

Reply:
The `outline-on-keyboard-focus` mixin relies on the `:focus-visible` selector being natively supported OR [the official polyfill](https://github.com/WICG/focus-visible) being loaded. In practice all browsers use the polyfill currently. That is expected to change in Chrome [soonish](https://chromestatus.com/features/5823526732824576).